unison-src/transcripts/blocks.output.md

```diff
@@ -253,9 +253,9 @@ ex n =
 
 ```ucm
 
-  These definitions depend on each other cyclically but aren't guarded by a lambda: zap19, zap210
+  The expression in red needs the {SpaceAttack} ability, but this location does not have access to any abilities.
+  
       5 |   zap1 = launchMissiles "neptune" + zap2
```

Comment:
I was puzzled about why this transcript output changed. See this comment if you're curious. The new output is correct: https://github.com/unisonweb/unison/pull/1732/commits/608ad4830c2059c6f21b79d65089148219009f47